cmd/apiserver_receive_adapter/main.go

```diff
@@ -89,9 +88,8 @@ func main() {
 	if err != nil {
 		panic(fmt.Sprintf("Error processing env var: %s", err))
 	}
-	// TODO move this util to pkg
-	// Convert base64 encoded json logging.Config to logging.Config.
-	loggingConfig, err := utils.Base64ToLoggingConfig(env.LoggingConfigBase64)
```

Comment:
looks like this file is the only user of `Base64ToLoggingConfig` (besides a test). Let's remove it ?

Reply:
This was removed in this PR already... I'm using some pkg utilities for doing that.